includes/helpers.php

```diff
@@ -5,9 +5,9 @@
 /**
  * Get all the checklist items that have values stored.
  *
- * @return array
+ * @return mixed
  */
-function get_wp_launch_checklist_values() : array {
+function get_wp_launch_checklist_values() : mixed {
```

Comment:
Should this be supported with a bump to the required PHP version [here](https://github.com/kanopi/wp-launch-checklist/blob/0a2b9573e5f0c026a41d2a5f817a53c310676adb/readme.txt#L7) and [here](https://github.com/kanopi/wp-launch-checklist/blob/0a2b9573e5f0c026a41d2a5f817a53c310676adb/wp-launch-checklist.php#L117)? Otherwise, it could lead to a type error under PHP 7.4 (...apologies if this will be covered at a later stage/before release).

Reply:
Thanks. Nice catch. Not sure why I thought mixed pseudo types were a 7.4 thing. Just going to bump the version requirement and use the proper union type instead.

Reply:
Also planning to rework this plugin a bit to allow upload of a checklist items source file (following a template) and fallback to the config if empty to allow more customization of checklist items (+ maybe an option to export the current status or provide a custom API endpoint it can be obtained from). When I have some time of course. Internally we don't always need all the current available settings and sometimes need other specific ones.